examples/vue-kitchen-sink/src/stories/addon-contexts.stories.js

```diff
@@ -0,0 +1,144 @@
+import { storiesOf } from '@storybook/vue';
+import { withContexts } from '@storybook/addon-contexts/vue';
+
+// Example A: Simple CSS Theming
+const topLevelContexts = [
+  {
+    icon: 'sidebaralt',
+    title: 'CSS Themes',
+    components: ['div'],
+    params: [
+      {
+        name: 'Desert',
+        props: {
+          style: { color: 'brown', background: '#F4A261', height: '100vh', padding: '10px' },
+        },
+      },
+      {
+        name: 'Ocean',
+        props: {
+          style: { color: 'white', background: '#173F5F', height: '100vh', padding: '10px' },
+        },
+        default: true,
+      },
+    ],
+  },
+];
+
+const storyLevelContexts = [
+  {
+    title: 'CSS Themes',
+    params: [
+      {
+        name: 'Forest',
+        props: {
+          style: { color: 'teal', background: '#00b894', height: '100vh', padding: '10px' },
+        },
+      },
+    ],
+  },
+];
+
+const stories = storiesOf('Addon|Contexts', module).addDecorator(withContexts(topLevelContexts));
+
+stories.add(
+  'Simple CSS Theming',
+  () => ({
+    template:
+      "<span>I'm a children of the injected 'div' (where provides a theming context).</span>",
+  }),
+  {
+    contexts: storyLevelContexts,
+  }
+);
+
+// Example B: Language (Vue provide/inject API)
+const createContext = initialValue => {
+  const uid = `_${Date.now()}${Math.random()}`;
+  return {
+    Provider: {
+      name: `Context.Provider`,
+      props: ['value'],
+      provide() {
+        return this.value === undefined
+          ? undefined
+          : {
+              [uid]: () => this.value,
+            };
+      },
+      template: `
+        <div>
+          <slot />
+        </div>
+      `,
+    },
+    Consumer: {
+      name: `Context.Consumer`,
+      inject: {
+        [uid]: {
+          default: () => () =>
+            initialValue instanceof Object ? { ...initialValue } : { value: initialValue },
+        },
+      },
+      computed: {
+        value() {
+          return this[uid]();
+        },
+      },
+      template: `
+        <div>
+          <slot v-bind="value" />
+        </div>
+      `,
+    },
+  };
+};
```

Comment:
I'm thinking to make this util component a standalone package or as a helper in the addon.  I have google and look into NPM registry, few examples or ideal implementation (vue2.6 changed something so they seems a bit outdated)...

Reply:
I have published the latest implementation for this util component: [vue-create-context](https://github.com/leoyli/vue-create-context) with some minor improvements.  But I think we can just merge this PR since it is not super long and working well?!